torch/_dynamo/symbolic_convert.py

```diff
@@ -1600,6 +1605,19 @@ def BEFORE_WITH(self, inst):
         self.push(exit)
         self.push(ctx.enter(self))
 
+    def append_prefix_inst(self, inst):
```

Comment:
Do we need to enforce the ordering of the inst in here? Or it should always be ok since we add things as we read the original frame?

Reply:
We add things as read in the original frame, and any instructions that jump necessarily come after the prefix, and they shouldn't jump to before the prefix.